### ISSUE TITLE
Transparency Issue / Colors

### ISSUE THREAD

**Issue by radioshadow** [JosephS068/Donkey-Konga-2-Texture-Editor#2]
Thanks for making this tool. Editing and replacing images seems to work.

However, transparency colors seem to be having issues, and some colors are not importing correctly.

Here's the title being loaded in the tool (no change). Not sure why the transparency isn't displaying correctly.
![1](https://github.com/user-attachments/assets/7fdd6b6b-75c6-450f-8be7-cb810a418031)

I make a small to insert some transparency in the corner.
![2](https://github.com/user-attachments/assets/07171213-39ca-443a-bf98-ef26316b8f88)

I load it back in the tool.
![3](https://github.com/user-attachments/assets/078f29da-76b2-46cf-a591-13ee1a894a3b)
Some of the colors are not right. The yellow is now a light orange in the name.

Here it is in the game.
![4](https://github.com/user-attachments/assets/fe11edee-1d95-41aa-8b80-a7cd17ce5248)

Here is what the title should look like in the original.
![5](https://github.com/user-attachments/assets/8d3f1633-44a4-4156-b5be-01827a4b47aa)

If you could fix this issue, that would be fantastic!

**Comment by JosephS068** [comment]
Yeah I tried to add a simple transparency background for images, but that was actually getting written to the file. So exporting and then reimporting was going to be a problem, I had always been overwriting the files with brand new images. This should now be addressed.  

[Fixed Commit](https://github.com/JosephS068/Donkey-Konga-2-Texture-Editor/commit/1c313e0c95f9c79974e9537850ae8f8bc94a2285)

**Comment by JosephS068** [comment]
For now transparent will be a black pixel. Not sure if that plays nice with the other textures appearance wise, but that should hopefully get you unstuck for now.
